### PR TITLE
Fix for issue #638

### LINKED ISSUE #638: mscolab chat upload/download  did not use configured "filepicker_default": "fs"
This is similar to
https://github.com/Open-MSS/MSS/issues/636

if we configure in mss_settings.json the "fs" filepicker this is currently for mscolab chat upload/download ignored

To verify this you need a config file mss_settings.json to be stored in $HOME/.config/mss

{
 "filepicker_default": "fs"
}

mss_qt.get_open_filename needs to be used

source to look at
https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/mscolab_project.py

please check if the test needs a fix too

### PR DESCRIPTION
**Pull request by aravindm711**

### CODE REVIEW

**Comment by aravindm711** [comment]
The tests seems to be working fine with the changes.

File: mslib/msui/mscolab_project.py [hunk 1]
```diff
@@ -231,14 +231,12 @@ def toggle_preview(self):
             self.previewBtn.setText("Write")
 
     def handle_upload(self):
-        img_type = "Image (*.png *.gif *.jpg *jpeg *.bmp)"
-        doc_type = "Document (*.*)"
-        file_filter = f'{img_type};;{doc_type}'
-        file_path, file_type = QtWidgets.QFileDialog.getOpenFileName(self, "Select a file", "", file_filter)
-        if file_path == "":
+        file_filter = ["Image (*.png *.gif *.jpg *jpeg *.bmp)", "Document (*.*)"]
```
Comment (ReimarBauer) [inline mslib/msui/mscolab_project.py]: All these examples use ";;" check this with the docs what is the difference to a ","

https://stackoverflow.com/questions/1604440/how-to-set-selected-filter-on-qfiledialog

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please check meaning of delimiter change from ";;" to ","

File: mslib/msui/mscolab_project.py (hunk 1 above)
Comment (aravindm711) [reply mslib/msui/mscolab_project.py]: the file_filter required for QtWidgets.QFileDialog.getOpenFileName is in the format that involves separating filters with ';;' (https://doc.qt.io/archives/qt-4.8/qfiledialog.html#getOpenFileName), but fs.file_picker takes file_filter as a list, that is why the new file_filter is a list of filters.

Comment (aravindm711) [reply mslib/msui/mscolab_project.py]: there was an error with qt filepicker after the changes. I have fixed them now for both fs and qt.

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**